src/App.js

```diff
@@ -1,27 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import Screen from './components/screen';
+import ButtonsContainer from './components/buttons-container';
+import Button from './components/button';
+
+// const buttonList = [
+// 'AC', '+/-', '%', '/', '7', '8', '9', 'X', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=',
+// ];
 
 function App() {
```

Comment:
- [ ] From the project requirement your `App.js` component should be **class-based**. Please fix it in order to make your project follow the requirement which is important :+1: